api/nodes/serializers.py

```diff
@@ -273,6 +278,30 @@ class NodeDetailSerializer(NodeSerializer):
     id = IDField(source='_id', required=True)
 
 
+class NodeForksSerializer(NodeSerializer):
+
+    category_choices = Node.CATEGORY_MAP.items()
```

Comment:
`category_choices` and `category_choices_string` *will* be inherited from `NodeSerializer`, but they won't be in scope for the `category` field. Therefore, you'll have to access them off of `NodeSerializer`.

```python
category = ser.ChoiceField(read_only=True, choices=NodeSerializer.category_choices, ...)
```